Logbook.org

```diff
@@ -0,0 +1,293 @@
+#+title: Logbook
+#+author: @CFGrote:github.com
+#+date: 20241212
+
+* Dec. 10 2024
+   :PROPERTIES:
+   :CUSTOM_ID: dec.-10-2024
+   :ID:       af2db540-6116-4fb3-9f02-33368cb8456a
+   :END:
+** Run 1
+:PROPERTIES:
+:ID:       ea102cdd-2c30-4892-91db-67a39d9e906d
+:END:
+*** Testing all queries from remote via vpn
+:PROPERTIES:
+:ID:       1749151c-81d9-4d5d-9e3c-d2dd8b21d95d
+:END:
+- Servers: ontop, fuseki
+- number of query repeats: 30
+- client ip: 128.176.254.12
+- client hostname: micropop046
+- client software: apache-jena-5.2.0/bin/rsparql
+- rsparql version reported as 4.2.0
+- connection: vpn to uni-muenster
+- note: all queries were run once about 30min before. Noting reduced
+  query time on the 00-construct and other queries. Caching?
+
+*** Workflow
+:PROPERTIES:
+:ID:       dcc3a8d5-8c2a-49af-8a58-ac1ef814e72a
+:END:
+- executing all queries
+- Connected to Münster VPN
+- Server host: 10.14.28.137
+- checked all queries can be executed.
+- query 09 failed, moved it to [[queries/query_foundry]].
+- observed that queries run after the test respond faster, suspect
+  there is some caching going on.
+- results are saved in [[file:/queries/saved_runs/20241210T091800]]
+
+*** Shell commands
+:PROPERTIES:
+:ID:       ae012b94-04af-4027-82db-8161fc46d53d
+:END:
+shell command for ontop:
+
+#+begin_example shell
+  cd queries
+  for q in ??-*.rq; do ./timer.sh $query http://128.176.233.7:8080/sparql ontop 30
+#+end_example
+
+shell command for fuseki
+
+#+begin_example shell
+  cd queries
+  for q in ??-*.rq; do ./timer.sh $query http://128.176.233.7:3030/OME/sparql ontop 30
+#+end_example
+
+*** Results
+:PROPERTIES:
+:ID:       79eb7d26-fc85-4802-936a-b9a9bccf2f62
+:END:
+[[file:queries/saved_runs/20241210T091800/facet_walltime.png]]
+
+*** Discussion
+:PROPERTIES:
+:ID:       9fe63540-6427-44bc-8e10-07d53b0e0805
+:END:
+In all queries, ontop's query response time is faster than fuseki. Must be aware of caching, hence should try other query submission strategy (-> see Run 2, Run 3)
+
+** Run 2: Loop over 10 queries, 2 servers, 10 rounds in that order
+:PROPERTIES:
+:ID:       825c8799-bb59-4b89-aa22-1b4b2d1279c8
+:END:
+By changing the order with respect to Run1, we try to eliminate the effect of caching, as each server never sees the same two queries immediately one after the other, wherea
```

Comment:
grammar:
wherea -> whereas
and erase line break

Reply:
done in 32df07f